examples/create_bids_folder.py

```diff
@@ -1,7 +1,7 @@
 """
-==========================================
-Creating BIDS-compatible folders and files
-==========================================
+==============================================
+01. Creating BIDS-compatible folders and files
```

Comment:
```suggestion
01. Creating BIDS-compatible folder names and filenames
```

? I think it's a bit misleading the title -- because what you get as output will not pass the validator